src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,39 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) {
+    if (typeof obj !== "object") {
+        throw "Please provide valid object";
+    }
+
+    if (obj === {}) {
+        return {};
+    }
```

Comment:
Because of how equality works in JS, this would always be false and this block would never run. 
`{} === {}` is always `false`